### ISSUE TITLE
Pull request title not being updated as new versions are released

### ISSUE THREAD

**Issue by timja** [updatecli/updatecli#930]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

1.4.8 was the first version, but a few more versions were released before a reviewer got to it.

the PR description was updated by update cli, but PR title was never updated

![image](https://user-images.githubusercontent.com/21194782/195583760-5de08c62-9026-447a-ba61-887da9448143.png)

https://github.com/hmcts/cnp-flux-config/pull/18147

### Expected Behavior

title and description should be in sync

### Steps To Reproduce

https://github.com/hmcts/cnp-flux-config/blob/master/.github/workflows/updatecli.yaml

https://github.com/hmcts/cnp-flux-config/blob/master/updatecli/weekly.d/jenkins-packer.yaml

Basically just release a new version of something and PR title and description should be updated

### Environment

```markdown
- OS: Ubuntu 
- updatecli: v0.33.3
```

**Comment by olblak** [comment]
This seems issue seems duplicated with https://github.com/updatecli/updatecli/issues/866

**Comment by timja** [comment]
Thanks I missed that, I searched pull request, not PR and didn't see it when I glanced through issues

Looks to be the same
